tests/pytest/test_tdscf_products.py

```diff
@@ -0,0 +1,303 @@
+import numpy as np
```

Comment:
I wrote these to check my product functions. They probably don't need to be run on travis, as they do take a non negligible amount of time.